src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -3,19 +3,18 @@
 import java.awt.event.ActionEvent;
 import java.awt.event.ActionListener;
 
+import edu.kis.powp.jobs2d.Job2dDriver;
 import edu.kis.powp.jobs2d.command.CommandFactory;
 import edu.kis.powp.jobs2d.command.SmallSquareFactory;
 import edu.kis.powp.jobs2d.command.SquareFactory;
 import edu.kis.powp.jobs2d.drivers.DriverManager;
-import edu.kis.powp.jobs2d.features.DrawerFeature;
-import edu.kis.powp.jobs2d.features.DriverFeature;
 import edu.kis.powp.jobs2d.magicpresets.FiguresJoe;
 
 public class SelectTestFigureOptionListener implements ActionListener {
-	private final String FIGURE1 = "1";
-	private final String FIGURE2 = "2";
-	private final String FIGURE3 = "3";
-	private final String FIGURE4 = "4";
+	public final String FIGURE1 = "1";
+	public final String FIGURE2 = "2";
+	public final String FIGURE3 = "3";
+	public final String FIGURE4 = "4";
```

Comment:
Constants should be used in TestJobs2dPattern to define the names of tests. (This is later used below, String command=e.getActionCommand();)